### DISCUSSION TITLE
issue with PHP On install of AIO

### DISCUSSION THREAD

**Discussion by Super-6-1** [nextcloud/all-in-one#4262] [Questions] [answered]
Working on getting AIO to run but it seems like The Database stopped running, Apache and Notification Push are waiting for nextcloud and in nextcloud i am getting this error 

PHP Notice:  ZipArchive::extractTo(): Write of 8192 bytes failed with errno=28 No space left on device in phar:///var/www/html/updater/updater.phar/lib/Updater.php on line 700

any idea what can cause this? using Unraid with Docker.

**Comment by szaimen** [comment, marked as answer]
Hi, it looks like your partition runs out of space...
